README.md

```diff
@@ -87,3 +87,6 @@ The ones we are using are:
   -N MAX-CONNECT     maximum number of allowed connections
   -S WORK-MEM        set amount of memory for sorts (in kB)
 ```
+
+Additionally, to get a better update-performance, we use
+the `max_wal_size=5GB` as run-time options. 
```

Comment:
Why not just list this with the options above? Only because this option isn't documented by `--help`?